paec.g4

```diff
@@ -33,14 +33,19 @@ clef
 keysignature
     :
     DOLLAR
-    ((SHARP | FLAT )    notename+
-    (LEFTBRACKET (SHARP | FLAT )? notename+ RIGHTBRACKET)? // rism [ ] sometimes appear, but are not documented
-    |
-    (LEFTBRACKET (SHARP | FLAT )? notename+ RIGHTBRACKET) // rism [ ] sometimes appear, but are not documented
+    (
+    (SHARP | FLAT)
+    ( notename+ | squarebracket | (notename+ squarebracket)+ notename* | (squarebracket notename+)+ squarebracket? )
```

Comment:
wouldn't this handle both `(notename+ squarebracket)+ notename*` and `(squarebracket notename+)+ squarebracket?` 
```
notename* (squarebracket notename+)+ squarebracket?

Reply:
`( notename* (squarebracket notename+)* squarebracket? )` seems to work too?

Reply:
Not really. The problem would be that $x without any note would also be allowed. Maybe we should avoid this?

Reply:
```
( notename+ | notename* (squarebracket notename+)* squarebracket? )
```
Looks good to me

Reply:
Push to the same branch. The pull request will then update with the changes

Reply:
I pushed this version to the branch square_brackets, to make it easier to test.
I also updated files in the folder pypaec (after compiling)

Reply:
The modified files are paec.g4, pypaec/pypaec.interp, pypaec/pypaecLexer.py,  pypaec/pypaecListener.py, pypaec/pypaecParser.py (in [7551e4e](https://github.com/rism-digital/paec-grammar/pull/1/commits/7551e4ec1817563bd995a2b57f026c47f8205364), below)

Reply:
Ah, ok. For some reason they weren't showing up earlier, but they are now.